site/class-2-style/Senmetsu/index.js

```diff
@@ -0,0 +1,56 @@
+
+var width = 500,
+    height = 500,
+    radius = Math.min(width, height) / 2,
+    colors = ['hsl(45, 99%, 59%)',
+      'hsl(96, 99%, 59%)',
+      'hsl(148, 99%, 59%)',
+      'hsl(199, 99%, 59%)',
+      'hsl(250, 99%, 59%)',
+      'hsl(302, 99%, 59%)',
+      'hsl(353, 99%, 59%)'],
+
+    color = d3.scaleOrdinal().range(colors),
+    arc = d3.arc().outerRadius(radius - 16).innerRadius(radius - 128),
+    pie = d3.pie().sort(null).value(population),
+
+    svg = d3.select('body').append('svg')
+    .attr('width', width)
+    .attr('height', height)
+    .attr('viewBox',[0, 0, width, height].join(' ')).append('g')
+    .attr('transform', 'translate(' + width / 2 + ',' + height / 2 + ')');
+
+d3.csv('index.csv', type, render)
+
+function render(b, c) {
```

Comment:
This looks great but you should update your one-letter-variables!

Reply:
Tip: you can do that by clicking on the pencil icon here on the top-right of a file.